algebird-util/src/main/scala/com/twitter/algebird/util/summer/HeavyHittersCachingSummer.scala

```diff
@@ -107,7 +101,7 @@ class ApproxHHTracker(hhPct: HeavyHittersPercent, updateFreq: UpdateFrequency, r
 
     if (hh.containsKey(item)) {
       val v = hh.get(item)
-      val newItemCount = +1L
+      val newItemCount = v + 1L
```

Comment:
@ianoc can you double check this? This triggered a warning in scalac (which we ignored), but it looks like we intended to increment the count, but lost the `v`. What do you think?

I guess this means this cache was always pruning...

Reply:
ouch, yeah this change looks right/looks like a bug.